File: DesktopApp/Model/EducationalExperience.cs
using Newtonsoft.Json;
using System.Windows.Controls;

namespace ProfessorPerformanceEvaluation.Model
{
    public class EducationalExperience
    {
        [JsonProperty("idEducationalExperience")]
        public int IdEducationalExperience { get; set; }
        [JsonProperty("name")]
        public string Name { get; set; }

        public override string ToString()
        {
            return Name;
        }
    }
}

File: MobileApp/app/src/main/java/com/example/professorperformanceevaluation/model/EducationalExperience.java
package com.example.professorperformanceevaluation.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EducationalExperience {
    @SerializedName("idEducationalExperience")
    @Expose
    private int idEducationalExperience;
    @SerializedName("name")
    @Expose
    private String name;

    public EducationalExperience() {
    }

    public int getIdEducationalExperience() {
        return idEducationalExperience;
    }

    public void setIdEducationalExperience(int idEducationalExperience) {
        this.idEducationalExperience = idEducationalExperience;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}